script.js

```diff
@@ -0,0 +1,34 @@
+var app = new Vue({
+  el: "#app",
+  data: {
+    writedTask: "",
+    TodoId: 1,
+    items: [],
+  },
+  methods: {
+    addNewTask: function () {
+      if (this.writedTask === "") {
+        return items;
+      } else {
+        this.items.push({
```

Comment:
Instead of `push` and other array mutations, prefer using a declarative higher order function like:
```javascript
this.items = [...this.items, { item }]
```